docs/guides/tutorials/getting_started/validate_your_data.rst

```diff
@@ -0,0 +1,97 @@
+.. _tutorials__getting_started__validate_your_data:
```

Comment:
The content is still mostly the old "create a checkpoint" page. I renamed it since the tutorial audience may not know what checkpoint means, so "validation" seemed clearer.

Reply:
+1

Reply:
+1